data/yam/agama/auto/template.libsonnet

```diff
@@ -52,8 +54,9 @@ function(bootloader=true,
             onlyRequired: if software_only_required then true,
           }),
           [if product != '' then 'product']: {
-            [if registration_code_ha != '' then 'addons']: std.prune([
+            [if registration_code_ha != '' || packagehub then 'addons']: std.prune([
```

Comment:
s/packagehub/registration_packagehub/
(it is still a registration, but without reg code)